lms/templates/logout.html

```diff
@@ -5,30 +5,45 @@
 {% block title %}{% trans "Signed Out" as tmsg %}{{ tmsg | force_escape }} | {{ block.super }}{% endblock %}
 
 {% block body %}
-    {% if enterprise_target %}
-        {% comment %}
-            For enterprise SSO flow we intentionally drop learner's session.
-            We are showing this signin message instead of logout message
-            to avoid any confusion for learner in that case.
-        {% endcomment %}
-        <h1>{% trans "We are signing you in." as tmsg %}{{ tmsg | force_escape }}</h1>
+    {% if show_tpa_logout_link %}
+        <h1>{% trans "You have signed out." as tmsg %}{{ tmsg | force_escape }}</h1>
 
         <p style="text-align: center; margin-bottom: 20px;">
-            {% filter force_escape %}
-            {% blocktrans %}
-              This may take a minute. If you are not redirected, go to the home page.
+            {% blocktrans trimmed asvar sso_signout_msg %}
+              If you want to delete the SSO session, please {start_anchor}click here{end_anchor}.
```

Comment:
I do not like how this message is worded. What about 
`<anchor>Click here</anchor> to delete your single signed on (SSO) session`

Reply:
@schenedx updated message.